### DISCUSSION TITLE
Old CMakeModules hash

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#1375] [Announcements]
CMakeModules is pointing to a hash from September 2021, which means fixes like https://github.com/NOAA-EMC/CMakeModules/pull/62 are not currently incorporated.

**Comment by junwang-noaa** [comment]
@jkbk2004 FYI.

**Comment by jkbk2004** [comment]
Sure! we will give a try to update soon: hopefully in a week or so.
